frontend/src/views/ProfileView.vue

```diff
@@ -39,10 +39,12 @@ export default defineComponent({
 	async created() {
 		await useUserStore().refreshAllUsers();
 		const filteredUsers = useUserStore().allUsers.find((user: User) => Number(user.id) === Number(this.profile_id));
+
 		if (filteredUsers === undefined) {
 			this.$router.push('/profiles')
 		} else {
 			this.user = filteredUsers
+			console.log("avatar", this.user.avatar);
```

Comment:
this can be removed

Reply:
also, prettierjs for this file :P

Reply:
also, should this also be updated to use the `storeToRefs` thing ? (maybe that's another issue)